tests/commands/sgaTests.js

```diff
@@ -0,0 +1,126 @@
+var sga = require("__buttercup/classes/commands/command.sga.js");
+
+var fakeGroup;
+
+module.exports = {
+  setUp: function(cb) {
+    this.command = new sga();
+    (cb)();
```

Comment:
Only have eslint running source. Tried on tests and it threw hundreds of errors - will make that a task :)